visualization/babel.config.json

```
@@ -0,0 +1,15 @@
+{
+	"presets": [
+		[
+			"@babel/preset-env",
+			{
+				"targets": {
+					"node": "current"
+				},
+				"corejs": "3.0.0",
```

```suggestion
				"corejs": "3",
```

At least that's how I read the [documentation](https://babeljs.io/docs/en/babel-preset-env#corejs).